decimal/decimal_test.go

```diff
@@ -116,6 +116,17 @@ var correctnessSamples = []struct {
 	{"-1234567891234567890.0987654321987654321", "c7150113012345678912345678900987654321987654321d", false},
 }
 
+var correctnessDecodeSamples = []struct {
+	numString string
+	mpBuf     string
+	fixExt    bool
+}{
+	{"1e2", "d501fe1c", true},
+	{"1.1e31", "c70301e2011c", false},
+	{"13e-2", "c7030102013c", false},
+	{"-1e3", "d501fd1d", true},
```

Comment:
May you return values from tarantool as etalon response. It's hard to validate that `d501fe1c` is correct response

Reply:
I've added `TestUnmarshal_from_decimal_new` where we create a decimal inside Tarantool with eval:

`return require('decimal').new(str)`